mqsf/exceptions.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2023 SUSE LLC.  All rights reserved.
+#
+# This file is part of mqsf.
+#
+
+
+class MQSFException(Exception):
+    """
+    Base class to handle all known exceptions.
+
+    Specific exceptions are implemented as sub classes of MQSFError
+
+    Attributes
+
+    * :attr:`message`
+        Exception message text
+    """
+    def __init__(self, message):
+        self.message = message
+
+    def __str__(self):
+        return format(self.message)
+
+
+class MQSFCommandException(MQSFException):
+    """
+    Exception raised if an external command called via a Command
+    instance has returned with an exit code != 0 or could not
+    be called at all.
+    """
+
+
+class MQConnectionException(MQSFException):
+    """
+    Exception raised of connection to RabbitMQ server failed
```

Comment:
We could remove RabbitMQ reference to have the code MQ agnostic?

Reply:
Yep, fixed. It will also be dependent on the decision to have or not have a broker to connect to.